capella2polarion/converters/link_converter.py

```diff
@@ -380,11 +384,20 @@ def _sorted_unordered_html_list(
 
 def _resolve_attribute(
     obj: common.GenericElement, attr_id: str
-) -> common.ElementList[common.GenericElement] | common.GenericElement:
+) -> common.ElementList[common.GenericElement] | common.GenericElement | None:
     attr_name, _, map_id = attr_id.partition(".")
     objs = getattr(obj, attr_name)
     if map_id:
```

Comment:
```suggestion
    if map_id and objs is not None:
```
otherwise we would still have an exception, if we have a `map_id` wouldn't we?